### PR TITLE
add InheritedPropertyDescriptor unit tests

### PR DESCRIPTION
**Pull request by Epica3055**
related to #714

add InheritedPropertyDescriptor unit tests
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/11432)

### CODE REVIEW

File: src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/InheritedPropertyDescriptorTests.cs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+using System.ComponentModel;
+using System.ComponentModel.Design;
+using System.Drawing;
+
+namespace System.Windows.Forms.Design.Tests;
+
+public class InheritedPropertyDescriptorTests
+{
+    [Fact]
+    public void GetValue_ReturnsExpected()
```
Comment (SimonZhao888) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/InheritedPropertyDescriptorTests.cs:13]: Please rename these test cases name like : InheritedPropertyDescriptor_GetValue_ReturnsExpected.

File: src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/InheritedPropertyDescriptorTests.cs [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    {
+        using Control control = new();
```
Comment (SimonZhao888) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/InheritedPropertyDescriptorTests.cs:15]:
```
using Control control = new();
PropertyDescriptorCollection properties = TypeDescriptor.GetProperties(control);
PropertyDescriptor propertyDescriptor = properties.Find(nameof(Control.Size), false);
InheritedPropertyDescriptor inheritedPropertyDescriptor = new(propertyDescriptor, control);
```
These codes can be moved into a public method, and should we add a test method for the default value?

File: src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/InheritedPropertyDescriptorTests.cs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using System.ComponentModel.Design;
+using System.Drawing;
+
+namespace System.Windows.Forms.Design.Tests;
+internal static class ExtensionHelpers
+{
+    public static InheritedPropertyDescriptor GetInheritedPropertyDescriptor(this Control control, string property)
```
Comment (lonitra) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/InheritedPropertyDescriptorTests.cs:11]: Could we move this into a separate file and rename to InheritedPropertyDescriptorTestExtensions?

**Review by lonitra — APPROVED** [review]
👍

**Pull request merged**